purchase_request_to_procurement/models/purchase_request.py

```diff
@@ -73,6 +73,7 @@ def _prepare_procurement_order_values(
             'product_qty': self.product_qty,
             'product_uom': self.product_uom_id.id,
             'date_planned': self.date_required,
+            'request_id': self.request_id.id,
```

Comment:
One solution that comes to my mind is to move the definition of `request_id` field from `purchase_request_procurement` to `purchase_request`.